anatomy/tracks/concept-exercises.md

```diff
@@ -162,13 +174,15 @@ lasagna.remaining_minutes_in_oven(30)
 
 **Purpose:** Provide hints to a student to help them get themselves unstuck in an exercise.
 
+**Presence:** Required
+
 - If the student gets stuck, we will allow them to click a button requesting a hint, which will show the relevant part of file.
 - Hints should be bullet-pointed underneath headings.
 - The hints should be enough to unblock almost any student.
 - The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).
 - The hints may use code samples to explain concepts, but not to outline the solution. e.g. in a lists exercise they might show a snippet of how a certain list function works, but not in a way that is directly copy/pasteable into the solution.
-- General hints about the exercise can appear under the `## General` heading.
-- Task-specific hints should appear underneath headings that match their task heading in the `instructions.md` (e.g. `## 2. Do Y`).
+- General hints about the exercise can appear as a Markdown list under the `## General` heading.
+- Task-specific hints should appear as a Markdown list underneath headings that match their task heading in the `instructions.md` (e.g. `## 2. Do Y`).
```

Comment:
I just noticed that we hadn't specified this, even though we depend on this

Reply:
I think you should add another point as we rely on this programatically:

- If there are no General hints for no hints for a specific task, the headings should be omitted. Every heading **must** be following by a markdown list.